game.js

```diff
@@ -234,7 +234,8 @@ module.exports = function createGame(options) {
         if (onlyAiLeft()) {
             destroyGame();
         }
-        emitState(true);
+        emitState();
+        game.emit('statechange');
```

Comment:
Is it possible for `statechange` to be emitted twice now? Would that cause any issues?

Reply:
Since the emitting of "statechange" was such a rare usecase for emitState I just pulled it out (removing the need for emitState to take a parameter) and call it explicitly here.